final-project/core/frontend/app/src/app/app.component.ts

```
@@ -0,0 +1,19 @@
+  selector: 'app-root',
+  standalone: true,
+  imports: [CommonModule, RouterOutlet],
+  template: `
+    <router-outlet></router-outlet>
+  `,
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  constructor(themeService: ThemeService) {
+    // Just inject the service to initialize it
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)